crates/rome_cli/src/termination.rs

```diff
@@ -0,0 +1,61 @@
+use std::{
+    env::current_exe,
+    ffi::OsString,
+    fmt::{self, Debug, Formatter},
+};
+use thiserror::Error;
+
+/// Error message returned by the CLI when it aborts with an error
+#[derive(Error)]
+pub enum Termination {
+    /// Returned by the CLI when it is called with a subcommand it doesn't know
```

Comment:
We already mentioned that this error is returned by the CLI when we documented `Termination`
```suggestion
    /// Returned when it is called with a subcommand it doesn't know
```